p2p/validation/msg_validation.go

```diff
@@ -15,11 +16,19 @@ type MsgValidatorFunc = func(ctx context.Context, p peer.ID, msg *pubsub.Message
 
 func MsgValidation(runner ssv.Runner) MsgValidatorFunc {
 	return func(ctx context.Context, p peer.ID, msg *pubsub.Message) pubsub.ValidationResult {
-		ssvMsg, err := DecodePubsubMsg(msg)
+		signedSSVMsg, err := DecodePubsubMsg(msg)
```

Comment:
maybe this is a complaint to myself.. but we should have deleted MsgValidation since it is definitely not what the code does at the moment

Reply:
Sure, let's remove it in another PR